### DISCUSSION TITLE
Is it possible to add customized functions in Nebula Explorer

### DISCUSSION THREAD

**Discussion by zhelin** [vesoft-inc/nebula#4396] [General]
Hi

since we have several specific requirements that Nebula Explorer cannot matched, I wonder if there has ways to integrate customized functions into Explorer tool.

thanks.
helin

**Comment by wey-gu** [comment]
Dear @zhelin 

Would you mind sharing your specific requirements for explorer? 

Explorer is a preparatory offering compared to Nebula studio(open-source), thus we don't have a way to enable customizing on explorer. I put PM and dev of Explorer in cc to see their ideas, too :).

cc @mizy @MuYiYong 

BR//Wey
